src/components/shareable-url-modal/shareable-url-modal.js

```diff
@@ -0,0 +1,336 @@
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+import { toggleShareableUrlModal } from '../../actions';
+import modifiers from '../../utils/modifiers';
+import { isRunningLocally } from '../../utils';
+import { s3BucketRegions } from '../../config';
+
+import Button from '../ui/button';
+import CopyIcon from '../icons/copy';
+import Dropdown from '../ui/dropdown';
+import IconButton from '../ui/icon-button';
+import Input from '../ui/input';
+import LoadingIcon from '../icons/loading';
+import Modal from '../ui/modal';
+import MenuOption from '../ui/menu-option';
+
+import './shareable-url-modal.scss';
+
+const modalMessages = (status, info = '') => {
+  const messages = {
+    default:
```

Comment:
can the error messages be in the config.js?

Reply:
It doesn't feel like they belong there since they're so specific to this component.

Reply:
Sure, ok with that. It's just easier to manage when all the hardcoded content is in one place.